e2e/publish/src/publish-private-packages.spec.ts

```diff
@@ -302,7 +302,7 @@ describe("lerna-publish-private", () => {
         lerna notice cli v999.9.9-e2e.0
         lerna WARN Unable to determine published version, assuming "test-X" unpublished.
         lerna notice from-package No unpublished release found
-        lerna success No changed packages to publish 
+        lerna success publish No changed packages to publish
```

Comment:
This prefix in this message doesn't seem to make it more readable or useful. I think it's fine without the prefix:
`lerna success No changed packages to publish` 

The command name in the prefix is mostly useful for the more verbose log levels as it helps debug issues in Lerna. For the success case, I think we want the most readable and concise messages possible.